src/backend/api/meals.js

```diff
@@ -1,15 +1,195 @@
-const express = require("express");
+const express = require('express');
+const { sum, limit } = require('../database');
 const router = express.Router();
-const knex = require("../database");
+const knex = require('../database');
 
-router.get("/", async (request, response) => {
-  try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
-  } catch (error) {
-    throw error;
-  }
+//Returns  meals based on query
+//GET api/meals/
+router.get('/', async(request, response) => {
```

Comment:
This handler has a similar problem to what I outline in [this comment](https://github.com/negaren/meal-sharing/pull/1#discussion_r826793745).

Reply:
i fixed according to your suggestion